__std__/nix/toolchain/scripts/stylish-haskell-checker.nix

```diff
@@ -0,0 +1,54 @@
+{ inputs, cell }:
+
+cell.packages.todo-derivation
+
+# TODO(std) need stylish-haskell for this
```

Comment:
A bunch of these you say you can't do yet, but you have todo derivations for e.g. stylish-haskell, so shouldn't you actually be able to define these and have them evaluate, just with the wrong tools being used?